kafka-on-ubuntu/azuredeploy.json

```diff
@@ -185,14 +197,17 @@
           },
           "subnetPrefix": {
             "value": "[parameters('subnetPrefix')]"
+          },
+          "tshirtSizes": {
+            "value": "[concat('List of sizes because of test for Unreferenced variable in CI - it is not needed for proper function.', variables('tshirtSizeS'), variables('tshirtSizeM'), variables('tshirtSizeL'))]"
```

Comment:
This isn't going to work because your t-shirt size variables are json objects... they must be strings or arrays.